qemu/x86_64/Makefile

```diff
@@ -25,7 +27,7 @@ fetch: getkernel geturoot
 
 getkernel:
 	rm -rf linux
-	git clone --depth=1 -b v5.10 https://github.com/torvalds/linux
+	git clone --depth=1 -b v5.8 https://github.com/torvalds/linux
```

Comment:
are you sure you want the older kernel?

Reply:
Hi Minnich, I prefer v5.10 too, but there are issues to be solved (.config settings), and make it fail at build. v5.8 is proven to be work fine. I'll try to upgrade the kernel back, and that's a goal.